tsconfig.json

```diff
@@ -60,6 +60,10 @@
       "featherlight": [
         "lib/featherlight/featherlight.js",
         "COMMENT"
+      ],
+      "node-forge": [
```

Comment:
Sorry for this custom lib include mechanism, I'm happy to help if it gets confusing.

Reply:
I figured out how it works so I guess it's not bad at all :)